core/src/main/java/io/confluent/kafka/schemaregistry/leaderelector/kafka/SchemaRegistryCoordinator.java

```diff
@@ -172,10 +187,15 @@ protected Map<String, ByteBuffer> onLeaderElected(
     }
 
     // Compute the leader as the leader-eligible member with the "smallest" (lexicographically) ID.
-    // This doesn't guarantee a member will stay leader until it leaves the group, but should
-    // usually keep the leader assigned to the same member across rebalances.
+    // If the group has an existing leader, this leader takes precedence. If multiple group
+    // members are determined to be leaders, fall back to the member with the smallest ID.
```

Comment:
For my understanding, when would multiple member be leaders? I understand they might be eligible to be leader.

Reply:
this should never happen. it's there as a failsafe to reset the leader if multiple leaders are found.